src/autoyast-rnc/firewall.rnc

```diff
@@ -129,14 +129,10 @@ zones =
       zone_short? &
       zone_description? &
       zone_target? &
-      fwd_forward_ports? &
```

Comment:
Remember also to modify the schema accordingly

https://github.com/yast/yast-schema/commit/7d3744e785db5965e9af00087e80e240a1929004